pkg/common/common.go

```diff
@@ -23,12 +23,12 @@ const (
 
 // Images
 const (
-	VeleroImage          = "quay.io/konveyor/velero:latest"
+	VeleroImage          = "quay.io/konveyor/velero:oadp-1.2"
```

Comment:
don't actually need these. IIRC we just read from ENV first.. only if ENV is empty/malformed do these hardcoded backups take effect.

Reply:
def non issue.. just a note.

Reply:
@kaovilai I was just following what we did for oadp-1.2 branch. In any case, if it's harmless, lets just leave it for now.